### PR TITLE
feat: move automations to a single json file

### PR DESCRIPTION
**Pull request by gt-codes**

### CODE REVIEW

File: src/automations/index.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "description": "It's 💦 Water Cooler Wednesday 🥳 Wanna know what we're chatting about? Hop in the `#general` channel and let us know if you have an interesting opinion or something you want to share!",
+        "channel": "general",
+        "entityType": "EXTERNAL",
+        "entityMetadata": {
+            "location": "#general channel"
+        }
+    },
+    {
+        "type": "job",
+        "expression": "0 * * * MON",
```
Comment (DarrylBrooks97) [inline src/automations/index.json]:
```suggestion
        "expression": "0 8 * * MON",
```

File: src/automations/index.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { GuildScheduledEventCreateOptions } from 'discord.js';
+import { client } from '..';
+import { CONSTANTS } from '../constants';
+import automations from './index.json';
+
+export default function run() {
+    automations.forEach(({ type, ...details }) => {
+        if (type === 'job') {
+            const callback = require(`./${details.callback}`);
+            schedule(details.expression, callback.default);
```
Comment (DarrylBrooks97) [inline src/automations/index.ts]:
```suggestion
            schedule(details.expression, callback.default,{
                timezone: 'America/Chicago',
            });
```

File: src/automations/index.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                ({ name }) => name === automations[0].channel,
+            )?.id;
+
+            const options = {
+                ...details,
+                ...CONSTANTS.DEFAULT_EVENT_OPTIONS,
+                channel,
+            } as GuildScheduledEventCreateOptions;
+
+            schedule(details.expression, () => guild?.scheduledEvents.create(options));
```
Comment (DarrylBrooks97) [inline src/automations/index.ts]:
```suggestion
            schedule(details.expression, () => guild?.scheduledEvents.create(options),{
                timezone: 'America/Chicago',
            });
```

File: src/automations/create-gnf-thread.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { formatISO } from 'date-fns';
+import { TextChannel } from 'discord.js';
+import { client } from '..';
+import { CONSTANTS } from '../constants';
+
+export default async function run() {
+    const date = formatISO(new Date(), { representation: 'date' });
+    const guild = client.guilds.cache.get(CONSTANTS.GUILD_ID);
+    const channel = guild?.channels.cache.get(CONSTANTS.GENERAL_CHANNEL_ID) as TextChannel;
+    console.log('works !');
```
Comment (DarrylBrooks97) [inline src/automations/create-gnf-thread.ts]:
```suggestion
```
I forgot to delete this

**Review by DarrylBrooks97 — CHANGES_REQUESTED** [review]

File: src/automations/index.ts (hunk 2 above)
Comment (blaccsmith) [reply src/automations/index.ts:11]: I'll add it to the default event options instead

File: src/automations/index.ts (hunk 3 above)
Comment (blaccsmith) [reply src/automations/index.ts:26]: Repeat of above suggestion

**Review by blaccsmith — APPROVED** [review]

File: src/automations/index.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        "type": "job",
+        "expression": "0 8 * * WED",
+        "callback": "create-wcw-thread"
+    },
+    {
+        "type": "job",
+        "expression": "0 8 * * FRI",
+        "callback": "create-gnf-thread"
+    }
+]
```
Comment (Genyus) [inline src/automations/index.json:39]:
```suggestion
{
  "events": [
    {
      "expression": "0 0 * * FRI",
      "name": "Good News Friday 🥳",
      "description": "It's Good News Friday and we want to celebrate your wins this week! Come share your accomplishments with us!",
      "channel": "moderator-only",
      "entityType": "EXTERNAL",
      "entityMetadata": {
        "location": "#general channel"
      }
    },
    {
      "expression": "0 8 * * TUE",
      "name": "💦 Water Cooler Wednesday",
      "description": "It's 💦 Water Cooler Wednesday 🥳 Wanna know what we're chatting about? Hop in the `#general` channel and let us know if you have an interesting opinion or something you want to share!",
      "channel": "general",
      "entityType": "EXTERNAL",
      "entityMetadata": {
        "location": "#general channel"
      }
    }
  ],
  "jobs": [
    {
      "expression": "0 8 * * MON",
      "callback": "verify-wcw-topic"
    },
    {
      "expression": "0 8 * * WED",
      "callback": "create-wcw-thread"
    },
    {
      "expression": "0 8 * * FRI",
      "callback": "create-gnf-thread"
    }
  ]
}
```
You can then access `automations.jobs` and `automations.events` more easily

Comment (gt-codes) [reply src/automations/index.json:39]: Addressed and pushed

**Pull request merged**